modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -6,62 +6,125 @@
 [id="distr-tracing-tempo-config-query_{context}"]
 = Query configuration options
 
-Query is a service that retrieves traces from storage and hosts the user interface to display them.
+Queries are managed in the {TempoShortName} by using two of its components, the querier and query frontend, both of which are configurable.
 
-.Parameters used by the {TempoOperator} to define Query
+The querier component finds the requested trace ID in the ingesters or back-end storage. Depending on the set parameters, the querier component can query both the ingesters and pull bloom or indexes from the back end to search blocks in object storage. The querier component exposes an HTTP endpoint at `GET /querier/api/traces/<traceID>`, but it is not expected to be used directly. Queries must be sent to the query frontend.
+
+.Configuration parameters for the querier component
 [options="header"]
-[cols="l, a, a, a"]
 |===
-|Parameter |Description |Values |Default value
-
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
-|
+|Parameter |Description |Values
+
+|`nodeSelector`
+|The simple form of the node-selection constraint.
+|type: object
+
+|`replicas`
+|The number of replicas to be created for the component.
+|type: integer; format: int32
+
+|`tolerations`
+|Component-specific pod tolerations.
+|type: array
 |===
 
+The query frontend component is responsible for sharding the search space for an incoming query. Traces are exposed via a simple HTTP endpoint: `GET /api/traces/<traceID>`. Internally, the query frontend component splits the `blockID` space into a configurable number of shards and then queues these requests. The querier component connects to the query frontend component via a streaming gRPC connection to process these sharded queries.
```

Comment:
Maybe "The query frontend exposes traces" instead of  "Traces are exposed"?

Reply:
Great suggestion. Thank you. Applying.